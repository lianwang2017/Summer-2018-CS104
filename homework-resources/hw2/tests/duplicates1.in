1 1 3 1 3 3 3 4 5 4 4 7
9 9 8 6
