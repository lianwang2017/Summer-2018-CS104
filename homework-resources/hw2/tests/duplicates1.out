1 3 1 3 4 5 4 7 9 9 8 6 
